docs/contributing.rst

```diff
@@ -0,0 +1,18 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. include:: ../CONTRIBUTING.rst
```

Comment:
Does this work? I thought sphinx doesn't let you include from outside the source tree?

Reply:
Yes. Proff: https://raw.githubusercontent.com/apache/airflow/master/docs/changelog.rst
http://wretched-hope.surge.sh/contributing.html

Reply:
Sphinx didn't allow to reference to external file, but it allow to import external file.